tests/pkcli/warning_test.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
This should be in test_data/tests/3_test.py. It should then have a log. It should actually be an asyncio example:

```
def test_coroutine_never_awaited():
    async def _test():
        asyncio.sleep(0)
    asyncio.run(_test)
```

Something like that.